### DISCUSSION TITLE
Test IntelliJ IDEA 2024.2 RC with Android Plugin

### DISCUSSION THREAD

**Discussion by twistios** [fast4x/RiMusic#3020] [General]
I have had problems with the debugger in Android Studio, even in the pre-release version (scanning for emulated method breakpoints did get stuck and could also not be canceled properly). This issue seems to be resolved in IntelliJ Backend already in the release canidate.

**Comment by aneesh1122** [comment]
I've updated android studio to ladybug but this annoying debug apk crash is still there

**Reply by twistios** [reply]
I am not talking about any Ladybug version, my comment was about IntelliJ IDEA 2024.2 RC (which is based on IntelliJ Platfrom which is also the backend of AndroidStudio) which is a "prerelease" of IDEA, as Ladybug was not helping with my issue. You can install Android Support into IDEA such that it is very similar to how Android Studio works.
